### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by Inkota**

### CODE REVIEW

**Comment by 0x346e3730** [comment]
Can you please fix the merge conflicts ? I've changed it to merge into development branch and not production.

**Comment by Inkota** [comment]
Idk if I can change commiting direction (can't find this function). But now all strings are translated and there shouldn't be any conflicts (:

File: JM/COT/languagecore/stringtable.csv [hunk 1]
```diff
@@ -1,181 +1,181 @@
...
+STR_COT_PLAYER_MODULE_RIGHT_PLAYER_IDENTITY_HEADER_DESC_DISABLED,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected,Отображается только если выбран 1 игрок,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected,Visible uniquement lorsque seulement 1 joueur est sélectionné,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected,Can only see when 1 player is selected
+STR_COT_PLAYER_MODULE_RIGHT_PLAYER_IDENTITY_NAME,Name:,Name:,Name:,Name:,Имя:,Name:,Name:,Name:,Name:,Nom:,Name:,Name:,Name:,Name:
+STR_COT_PLAYER_MODULE_RIGHT_PLAYER_IDENTITY_STEAMID,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam: ,Steam:
```
Comment (0x346e3730) [inline JM/COT/languagecore/stringtable.csv]:
```suggestion
STR_COT_PLAYER_MODULE_RIGHT_PLAYER_IDENTITY_STEAMID,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam:,Steam: 
```

File: JM/COT/languagecore/stringtable.csv [hunk 2]
```diff
@@ -1,181 +1,181 @@
...
+STR_COT_GENERIC_CREATE,Create,Create,Create,Create,Создать,Create,Create,Create,Create,Créer,Create,Create,Create,Create
+STR_COT_GENERIC_MAKE,Make,Make,Make,Make,Создать,Make,Make,Make,Make,Faire,Make,Make,Make,Make
+STR_COT_GENERIC_NO,No,No,No,No,Нет,No,No,No,No,Non,No,No,No,No
+STR_COT_GENERIC_OFF,off,off,VYP,AUS,выкл,WYŁ.,off,NO,NO,NON,關,オフ,off,关
 STR_COT_GENERIC_OFF_C,OFF,OFF,VYP,AUS,ВЫКЛ,WYŁ.,OFF,NO,NO,NON,關,オフ,OFF,关
-STR_COT_GENERIC_ON,on,on,ZAP,AN,ВКЛ,WŁ.,on,SÌ,SÍ,OUI,開,オン,on,开
+STR_COT_GENERIC_ON,on,on,ZAP,AN,вкл,WŁ.,on,SÌ,SÍ,OUI,開,オン,on,开
 STR_COT_GENERIC_ON_C,ON,ON,ZAP,AN,ВКЛ,WŁ.,ON,SÌ,SÍ,OUI,開,オン,ON,开
 STR_COT_GENERIC_SPAWN,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn
 STR_COT_GENERIC_YES,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Oui,Yes,Yes,Yes,Yes
```
Comment (0x346e3730) [inline JM/COT/languagecore/stringtable.csv]: Missing translation

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]
You're missing one translation and have extra spaces on another one

File: JM/COT/languagecore/stringtable.csv (hunk 2 above)
Comment (Inkota) [reply JM/COT/languagecore/stringtable.csv]: I can't find what I missed here, everything seems ok to me

Comment (0x346e3730) [reply JM/COT/languagecore/stringtable.csv]: STR_COT_GENERIC_YES is still "Yes" for russian

**Review by 0x346e3730 — APPROVED** [review]
Thank you

**Pull request merged**